assets/src/date/isToday.js

```diff
@@ -28,8 +33,8 @@ import moment from 'moment-timezone';
  * @return {boolean} If date matches today it will be true
  */
 export function isToday(date) {
-  const displayDate = moment.isMoment(date) ? date : moment.parseZone(date);
-  const today = moment().startOf('day');
+  const displayDate = getMoment(date);
+  const today = moment.utc().startOf('day');
```

Comment:
👍  this, i think, fixes something that I totally didn't notice on the first round of date utils for formatting.  Was doing some testing with manipulating modified dates for stories w/ different wordPress timezones + setting my browser timezone with an extension to be different and this branch fixes an issue I just saw when testing `main` for a sanity check where when modified date is today the browser date was getting in the way of the relative time formatted. So this is great!  

Dates are always tricky, dates in browser PLUS dates in wordPress is extra tricky.